libs/cms/src/lib/generated/contentfulTypes.d.ts

```diff
@@ -699,6 +699,12 @@ export interface IFrontpageFields {
   /** Title */
   title: string
 
+  /** Heading */
```

Comment:
Ran `yarn nx run api:contentful-types`.
This should be kept up to date if possible